vendor/kops-1.12/Makefile

```diff
@@ -3,6 +3,8 @@ export MAJOR_VERSION = 1.12
 export PACKAGE_NAME = $(MASTER_PACKAGE_NAME)-$(MAJOR_VERSION)
 export PACKAGE_REPO_NAME = kops
 export DOWNLOAD_URL = https://github.com/kubernetes/kops/releases/download/$(PACKAGE_VERSION)/kops-$(OS)-$(ARCH)
+export PACKAGE_VERSION_PIN = $(MAJOR_VERSION).x
+export AUTO_UPDATE_ENABLED = false
```

Comment:
We want this true, no?

Reply:
No, 1.12 is well past end-of-life, no need to update it, and more likely to cause problems if we try.